app/ServiceRates/ServiceRateController.php

```diff
@@ -30,4 +34,25 @@ public function getServiceRates(
             $transformerService->setPaginator($paginator)->transformResources($serviceRates)
         );
     }
+
+    public function getMultiColliServiceRates(
+        JsonRequestValidator $jsonRequestValidator,
+        MultiColliShipmentRequest $request,
```

Comment:
Not sure about using the full-blown MultiColliShipment request validation here, since at the point of requesting rates there are still things to be filled in, such as items and customs details. The regular `/get-service-rates` endpoint also does not use the request validator of the regular `/shipments` endpoint. I'd say:
```suggestion
        Request $request,
```